src/timeline/js/directives/ruler.js

```diff
@@ -52,14 +52,19 @@ App.directive("tlScrollableTracks", function () {
       element.on("wheel",function (e) {
         if (e.ctrlKey) {
           e.preventDefault(); // Don't scroll like a browser
-          if (e.originalEvent.deltaY > 0) { // Scroll down: Zoom out
+          var delta = e.originalEvent.deltaY * (e.shiftKey ? -1 : 1);
+          if (delta > 0) { // Scroll down: Zoom out
             /*global timeline*/
             timeline.zoomOut();
           } else { // Scroll Up: Zoom in
             /*global timeline*/
             timeline.zoomIn();
           }
         }
+        else if (e.shiftKey) {
+          let current_scroll = $("#scrolling_tracks").scrollLeft()
```

Comment:
This still _also_ needs the `preventDefault()`:
```suggestion
          e.preventDefault();
          let current_scroll = $("#scrolling_tracks").scrollLeft()
```

Reply:
I think I'll apply this suggestion and merge, this works perfectly in my testing. It's a nice, focused PR so there's no sense in letting it languish here forever.